packages/create-plugin/templates/common/.config/Dockerfile

```diff
@@ -4,10 +4,11 @@ ARG grafana_image=grafana-enterprise
 FROM grafana/${grafana_image}:${grafana_version}
 
 ARG development=false
+ARG TARGETARCH
 
 {{#if hasBackend}}
 ARG GO_VERSION=1.21.6
-ARG GO_ARCH=amd64
+ARG GO_ARCH=${TARGETARCH:-amd64}
```

Comment:
Sorry. I've done a mistake in my last comment. I wanted to suggest adding this:
```yaml
        TARGETARCH: ${TARGETARCH:-amd64}
```
in the `docker-compose.yaml` instead of `dockerfile`.
The code as it is works, but imho we could put it in `docker-compose.yml` to for consistency, since the grafana version and other args are also being declared there.